### PR TITLE
sh -c the cat repo for yum

### LINKED ISSUE #14590: Docker Experimental Script Installs Docker 1.6.2 Stable on CentOS
When running `wget -qO- https://experimental.docker.com | sh ` on CentOS 7, this installs `Docker version 1.6.2, build ba1f6c3/1.6.2`. Is there something I'm missing for experimental release installation?

### PR DESCRIPTION
**Pull request by jessfraz**
fixes #14590

### CODE REVIEW

**Comment by jfrazelle** [comment]
ping @tianon

**Comment by calavera** [comment]
LGTM

**Comment by tiborvass** [comment]
I wonder if we could have put just `"` double quotes. Apart from that LGTM

**Comment by jfrazelle** [comment]
can try

**Comment by jfrazelle** [comment]
that worked i updated, no idea why i did not do that in the first place... feel like i should blame systemd

File: hack/install.sh [hunk 1]
```diff
@@ -238,14 +238,14 @@ do_install() {
 			;;
 
 		fedora|centos)
-			cat >/etc/yum.repos.d/docker-${repo}.repo <<-EOF
+			$sh_c "cat >/etc/yum.repos.d/docker-${repo}.repo <<-EOF
```
Comment (tianon) [inline hack/install.sh]:
```bash
$sh_c "cat >/etc/yum.repos.d/docker-${repo}.repo" <<-EOF
```

Then you could avoid having the heredoc inside quotes.

Comment (jfrazelle) [inline hack/install.sh]: genius

**Comment by jfrazelle** [comment]
updated

**Comment by tiborvass** [comment]
LGTM

File: hack/install.sh (hunk 1 above)
Comment (thaJeztah) [inline hack/install.sh]: @tianon do you come in bulk packages? I'd like to have some of you in my toolset for when I need to do shell scripting :smile:

**Pull request merged**